### PR TITLE
fix(ci): only track lockfile.json

### PR DESCRIPTION
**Pull request by vigoux**

### CODE REVIEW

File: .github/workflows/update-parsers-pr.yml [hunk 1]
```diff
@@ -35,6 +35,11 @@ jobs:
           cp lockfile.json /tmp/lockfile.json
           cat /tmp/lockfile.json | /tmp/jq --sort-keys > lockfile.json
 
+      - name: Commit changes
+        run: |
+          git add lockfile.json
+          git commit -m "Update lockfile.json"
+
```
Comment (theHamsta) [inline .github/workflows/update-parsers-pr.yml:42]: maybe we could also add a cron trigger? Don't know how to formulate it that it only get's executed on master.
```
on:  
  push:
  pull_request:
  schedule:
    - cron: '0 0 * * *'  # every day at midnight
```

Comment (theHamsta) [reply .github/workflows/update-parsers-pr.yml:42]: probably also unrelated to this PR.

Comment (vigoux) [reply .github/workflows/update-parsers-pr.yml:42]: Yeah I think it's unrelated, and I'd like to keep it like this for now, to synchronise updates with actual activity periods of the plugin.

Comment (theHamsta) [inline .github/workflows/update-parsers-pr.yml:42]: how does this change anything in the logic? Won't the create pull request action do another commit?

Comment (vigoux) [reply .github/workflows/update-parsers-pr.yml:42]: Apparently not, given the example showed in the `README`

Comment (theHamsta) [reply .github/workflows/update-parsers-pr.yml:42]: :shrug: ok

Comment (vigoux) [reply .github/workflows/update-parsers-pr.yml:42]: Lol yeah, let's try that, it wont hurt anyway :wink:

**Review by theHamsta — APPROVED** [review]
